docker-compose.yml

```diff
@@ -7,6 +7,7 @@ services:
             - 8000:8000
         volumes:
             - ./:/usr/src/app/
+            - ./db:/usr/src/app/db
```

Comment:
Is this necessary if `./:/usr/src/app/` is already specified?

Reply:
This makes sure the db folder is actually created, which settings.py now has for the default db folder

Reply:
In that case, the `.gitkeep` file is not necessary, right?

Reply:
I just tested this locally and the .gitkeep and the db folder are not necessary with this change in the docker compose. Either change can thus be removed

Reply:
The folder is necessary if you don't use Docker. Without it and the .gitkeep `./manage.py migrate` outside a container won't work.